source/ScrollSync/ScrollSync.js

```diff
@@ -67,5 +101,18 @@ export default class ScrollSync extends React.PureComponent {
       scrollTop,
       scrollWidth,
     });
+    this._elements.forEach(element => {
+      if (element != null && element.scrollToPosition)
+        element.scrollToPosition({scrollLeft, scrollTop});
+    });
+  }
+
+  _registerChild(element: ScrollableElement | null) {
+    if (element == null || !element.scrollToPosition) {
```

Comment:
nit: i'd even just check `!element`

Reply:
Will fix! Should we consider keeping the part with scrollToPosition check since that's the thing we actually need from the element:

```suggestion
    if (!element || !element.scrollToPosition) {
```